source/patterns/04-pages/research-article/research-article.twig

```diff
@@ -29,7 +29,11 @@
         },
         {
           template: 'organisms-content-header',
-          arguments: contentHeader,
+          arguments: contentHeader|merge({ compact: true }),
```

Comment:
This is a shame (especially as it has to be implemented by the user). Makes me think we do need to consider it part of the content header.

Reply:
Looking at eLife's version, the Content Header doesn't have any block-end margin. That could work here too, but means that we'd need to add some block-start spacing elsewhere...

Reply:
I agree this could be in the content header as well. It would simplify things.